src/main/java/team/chisel/client/render/SubmapManagerGlotek.java

```diff
@@ -0,0 +1,80 @@
+package team.chisel.client.render;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.renderer.RenderBlocks;
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.util.IIcon;
+import net.minecraft.world.IBlockAccess;
+
+import team.chisel.ctmlib.RenderBlocksCTM;
+import team.chisel.ctmlib.TextureSubmap;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+public class SubmapManagerGlotek extends SubmapManagerBase {
+
+    @SideOnly(Side.CLIENT)
+    private static class RenderBlocksCTMFullbright extends RenderBlocksCTM {
+
+        @Override
+        protected void fillLightmap(int bottomLeft, int bottomRight, int topRight, int topLeft) {
+            ao();
+            int maxLight = 0xF000F0;
+            super.fillLightmap(maxLight, maxLight, maxLight, maxLight);
+        }
+
+        @Override
+        protected void fillColormap(float bottomLeft, float bottomRight, float topRight, float topLeft, float[][] map) {
+            ao();
+            int color = 0xFFFFFF;
+            super.fillColormap(color, color, color, color, map);
+        }
+
+        @Override
+        public boolean renderStandardBlock(Block block, int x, int y, int z) {
+            boolean ret = super.renderStandardBlock(block, x, y, z);
+            this.enableAO = false;
+            return ret;
+        }
+
+        private void ao() {
```

Comment:
It is considered bad practice to use abreviations in method/class/field names, please use a more descriptive method name.